benchexec/tablegenerator/react-table/src/components/QuantilePlot.js

```diff
@@ -565,6 +565,14 @@ export default class QuantilePlot extends React.Component {
                 resultsTooltip,
                 this.state.isResultSelectionDisabled,
               )}
+              {renderResetButton(() =>
+                setURLParameter(
+                  Object.keys(this.state).reduce((res, param) => {
+                    res[param] = null;
+                    return res;
+                  }, {}),
```

Comment:
Sorry for the delay due to some unforeseen events on my side. 

So should it be explicitly specified like this ?
```js
{renderResetButton(() =>
  setURLParameter({
    plot: null,
    scaling: null,
    results: null,
    selection: null
    // any other URL parameters
  }),
)}
```
I did current implementation because if a parameter is added in future, then it would explicitly need to be specified here in reset button which can be overlooked. Also for renaming a parameter in url, it has to be done all over the code.
 
I was wondering of a way where this can be avoided, so maybe a URL param map object can be created where the param and url param names will be mapped and it can be used everywhere.
So if we want to rename then we just have to change the value at one place.
```js
const urlParamMap = {
  plot: 'plot',
  scaling: 'scaling',
  results: 'results',
  // other URL parameters  
};
// ... //

{renderSetting("Plot",this.state.plot,
                (ev) => setURLParameter({ [urlParamMap.plot]: ev.target.value }),
                this.plotOptions,
              )}
```

If you prefer specifying explicitly, I am okay with that. How should I proceed ?

Reply:
@PhilippWendler

Reply:
I have changed it to manual way. Please review it.

Reply:
Nice, thanks!